rust/protocol/src/session_cipher.rs

```diff
@@ -228,6 +240,10 @@ pub async fn message_decrypt_signal<R: Rng + CryptoRng>(
         )
         .await?
     {
+        log::warn!(
+            "Previously known identity key for while encrypting for recipient: {}",
```

Comment:
OK I see these are directly from AxolotlKit but I still think they are not even English. :)